target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1 @@
+C:\Users\Mitko\Maven\murabi\src\main\java\com\murabi\api\MurabiServlet.java
```

Comment:
[major] Don't commit anything from the target dir since it contains only items that are built by maven.